crates/sargon-uniffi/src/wrapped_radix_engine_toolkit/low_level/compiled_transaction_intent.rs

```diff
@@ -0,0 +1,47 @@
+use crate::prelude::*;
+use sargon::CompiledTransactionIntent as InternalCompiledTransactionIntent;
+
+#[derive(Clone, PartialEq, Eq, uniffi::Record)]
+pub struct CompiledTransactionIntent {
+    secret_magic: BagOfBytes,
+}
+
+impl From<InternalCompiledTransactionIntent> for CompiledTransactionIntent {
+    fn from(value: InternalCompiledTransactionIntent) -> Self {
+        Self {
+            secret_magic: value.bytes().into(),
+        }
+    }
+}
+
+impl CompiledTransactionIntent {
+    pub fn into_internal(&self) -> InternalCompiledTransactionIntent {
+        self.clone().into()
+    }
+}
+
+impl Into<InternalCompiledTransactionIntent> for CompiledTransactionIntent {
+    fn into(self) -> InternalCompiledTransactionIntent {
+        unsafe {
```

Comment:
`unsafe`?

Reply:
Changed to https://github.com/radixdlt/sargon/pull/241#discussion_r1806664802